cpp/ql/src/semmle/code/cpp/dataflow/TaintTracking2.qll

```diff
@@ -0,0 +1,13 @@
+/**
+ * Provides classes for performing local (intra-procedural) and
+ * global (inter-procedural) taint-tracking analyses.
+ *
+ * We define _taint propagation_ informally to mean that a substantial part of
+ * the information from the source is preserved at the sink. For example, taint
+ * propagates from `x` to `x + 100`, but it does not propagate from `x` to `x >
+ * 100` since we consider a single bit of information to be too little.
+ */
+module TaintTracking2 {
+  import semmle.code.cpp.dataflow.internal.TaintTrackingUtil
```

Comment:
Shouldn't this import happen in `dataflow2/TaintTrackingImpl.qll` (and same for `TaintTracking.qll`)?

Reply:
It probably should. That would be consistent with how we do it for `DataFlow`. I'll change that.